src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -44,6 +44,43 @@ const PopulationActions = styled.div(() => [
   tw`col-span-3 p-1 pl-6`,
 ]);
 
+// Define fields to conditionally appear based on selected scoring unit
+export const DefaultPopulationSelectorDefinitions = [
```

Comment:
Question: I don't see entry for measure population and its exclusion which is available only for ratio measures. Is it part of future stories?

Reply:
Good catch, though I believe you're thinking of CV measures. They'll be added in another story.

Reply:
Sorry yeah, CV measure. then we are good. Thanks!